modules/custom/az_person/templates/horizontal-tabs--node--az-person--group-tabs.html.twig

```diff
@@ -21,20 +21,20 @@
   {% endif %}
 {% endfor %}
 {% if tabElements %}
-  <div class="mt-4 mb-3 border rounded clearfix">
-    <ul class="nav nav-tabs bg-gray-200 az-horizontal-tabs rounded-top" role="tablist">
+  <div class="mt-4 mb-3 clearfix">
+    <ul class="nav nav-tabs az-horizontal-tabs rounded-top" role="tablist">
```

Comment:
```suggestion
    <ul class="nav nav-tabs rounded-top" role="tablist">
```

Reply:
See https://github.com/az-digital/az_quickstart/blob/dfeb608e44b7ae693dff31976574f5c61c018fb7/modules/custom/az_person/css/az_person.css#L13

Reply:
We are no longer using `a` tags, so I think we can remove this class.